src/test/java/com/wikia/webdriver/elements/mercury/pages/MainPage.java

```diff
@@ -1,17 +1,32 @@
 package com.wikia.webdriver.elements.mercury.pages;
 
 import com.wikia.webdriver.elements.common.Navigate;
+import com.wikia.webdriver.elements.mercury.components.Category;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
 
 public class MainPage extends WikiBasePageObject {
 
+  private Navigate navigate;
+
   public MainPage() {
     super();
+
+    navigate = new Navigate(driver);
   }
 
   public MainPage open() {
-    new Navigate(driver).toPage("/");
+    navigate.toPage("/");
+
+    return this;
+  }
+
+  public MainPage openRegularMainPage() {
+    navigate.toPage("/wiki/Mercury_automation_testing_Wikia");
```

Comment:
we do have, but since we created objects that represent whole page, we are going to keep links there